Source/WebKit/UIProcess/Inspector/WebInspectorUIProxy.h

```diff
@@ -118,6 +118,7 @@ class WebInspectorUIProxy
 
 #if ENABLE(INSPECTOR_EXTENSIONS)
     WebInspectorUIExtensionControllerProxy* extensionController() const { return m_extensionController.get(); }
+    RefPtr<WebInspectorUIExtensionControllerProxy> protectedExtensionController();
```

Comment:
The one above is const, let's match it. We wouldn't want to make adoption of the protected overload harder.